ddl/index.go

```diff
@@ -597,14 +598,17 @@ func (w *worker) onCreateIndex(d *ddlCtx, t *meta.Meta, job *model.Job, isPK boo
 	switch indexInfo.State {
 	case model.StateNone:
 		// none -> delete only
+		reorgTp := pickBackfillType(job)
+		if reorgTp.NeedMergeProcess() {
+			indexInfo.BackfillState = model.BackfillStateRunning
```

Comment:
Because the DML writes must be redirected to the temp index once the index is created, otherwise the lightning remote duplicate check does not work correctly.

Reply:
So we needn't do https://github.com/pingcap/tidb/pull/37709/files#diff-59a1574ea9d9b6b78f0bd21d69e88f3861fae42835b177362638bea861a8397eR713?